app/src/test/java/org/simple/clinic/summary/bloodpressures/newbpsummary/NewBloodPressureSummaryViewEffectHandlerTest.kt

```diff
@@ -0,0 +1,47 @@
+package org.simple.clinic.summary.bloodpressures.newbpsummary
+
+import com.nhaarman.mockito_kotlin.doReturn
+import com.nhaarman.mockito_kotlin.mock
+import com.nhaarman.mockito_kotlin.whenever
+import io.reactivex.Observable
+import org.junit.After
+import org.junit.Test
+import org.simple.clinic.bp.BloodPressureRepository
+import org.simple.clinic.mobius.EffectHandlerTestCase
+import org.simple.clinic.patient.PatientMocker
+import org.simple.clinic.util.scheduler.TrampolineSchedulersProvider
+import java.util.UUID
+
+class NewBloodPressureSummaryViewEffectHandlerTest {
+
+  private val bloodPressureRepository = mock<BloodPressureRepository>()
+  private val effectHandler = NewBloodPressureSummaryViewEffectHandler(
+      bloodPressureRepository = bloodPressureRepository,
+      schedulersProvider = TrampolineSchedulersProvider()
+  ).build()
+  private val testCase = EffectHandlerTestCase(effectHandler)
+  private val config = NewBloodPressureSummaryViewConfig()
+
+  @After
+  fun tearDown() {
+    testCase.dispose()
+  }
+
+  @Test
+  fun `when load blood pressures effect is received, then load blood pressures`() {
+    // given
+    val patientUuid = UUID.fromString("6b00207f-a613-4adc-9a72-dff68481a3ff")
+    val bloodPressure = PatientMocker.bp(
+        UUID.fromString("51ac042d-2f70-495c-a3e3-2599d8990da2"),
+        patientUuid
+    )
+    val bloodPressures = listOf(bloodPressure)
+    whenever(bloodPressureRepository.newestMeasurementsForPatient(patientUuid = patientUuid, limit = config.numberOfBpsToDisplay)) doReturn Observable.just(bloodPressures)
```

Comment:
The effect handler itself doesn't need the config since the number of BPs comes from the effect. To the effect handler, this config is irrelevant. In this particular test we should only care about the number of BPs to show from the effect, not where that value comes from.

Not necessary to fix this in this PR, we can change this in the next.

Reply:
I have fixed it in this PR. Instead of using config to get the `numberOfBpsToDisplay`, I have created a variable `numberOfBpsToDisplay ` in the test.